node_name1:
  ros__parameters:

    # 基本設定
    hz: 10              # ループ周期 [Hz]
    particle_num: 500   # パーティクルの個数 [-]
    max_particle_num: 600   #パーティクルの最大個数
    min_particle_num: 300   #パーティクルの最小個数
    move_dist_th: 0.025 # ロボットの移動開始判断用（スタートからの距離の閾値）[m]

    # 初期ポーズ関連
    init_x: 0.037751       # 初期位置 [m]
    init_y: 1.963176       # 初期位置 [m]
    init_yaw: -0.5     # 初期姿勢 [rad]
    init_x_dev: 0.80  # 初期位置xの標準偏差 [m]
    init_y_dev: 0.80  # 初期位置yの標準偏差 [m]
    init_yaw_dev: 0.80 # 初期姿勢の標準偏差 [rad]

    # リセット関連
    alpha_th: 0.0017        # リセットに関する平均尤度の閾値 [-]
    reset_count_limit: 5    # 連続リセットの回数の上限 [-]
    expansion_x_dev: 0.05   # 膨張リセットの位置xの標準偏差 [m]
    expansion_y_dev: 0.05   # 膨張リセットの位置yの標準偏差 [m]
    expansion_yaw_dev: 0.02 # 膨張リセットの姿勢の標準偏差 [rad]

    # センサ関連
    laser_step: 10           # 何本ずつレーザを見るか(全1080本)[-]
    sensor_noise_ratio: 0.03 # 距離に対するセンサノイズ比 [-]

    # --- 柱に関する角度範囲の配列 [rad] ---
    # 以下が無視される
    # [0] < angle < [1], [2] < angle
    # [num] -> num:index
    ignore_angle_range_list: [-0.80, -0.62, 0.63, 1.20, 2.24, 2.74, 3.75, 3.93]
    
    # フラグ
    flag_init_noise: true # 初期位置にノイズを加えるか
    flag_broadcast: true  # tf情報を含むbagファイルを使うときはfalse
    flag_reverse: false   # 初期姿勢を逆にするか
    is_visible: true      # パーティクルクラウドを表示（パブリッシュ）するか

    # ===== クラスOdomModel =====
    ff: 0.17   # 直進1[m]で生じる道のりのばらつきの標準偏差 [m]
    fr: 0.0005 # 回転1[rad]で生じる道のりのばらつきの標準偏差 [m]
    rf: 0.13   # 直進1[m]で生じる向きのばらつきの標準偏差 [rad]
    rr: 0.2    # 回転1[rad]で生じる向きのばらつきの標準偏差 [rad]